community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigrator.java

```diff
@@ -216,6 +210,9 @@ public void migrate( File storeDir, File migrationDir, MigrationProgressMonitor.
                     lastTxId, lastTxInfo.checksum(), lastTxLogPosition.getLogVersion(),
                     lastTxLogPosition.getByteOffset(), progressMonitor, oldFormat, newFormat );
         }
+        // update necessary neostore records
+        LogPosition logPosition = readLastTxLogPosition( migrationDir );
+        updateOrAddNeoStoreFieldsAsPartOfMigration( migrationDir, storeDir, versionToMigrateTo, logPosition );
```

Comment:
No, this is here because it needs to happen before count migration, and thus can not be in move phase anymore :)